### ISSUE TITLE
Compatibility with Elasticsearch 6.6.1

### ISSUE THREAD

**Issue by HernanMora** [opendistro-for-elasticsearch/security-ssl#3]
Hi!
I want to test the plugin with Elasticsearch 6.6.1 but I am getting the following error:

```
[INFO] Scanning for projects...
[INFO] ------------------------------------------------------------------------
[INFO] Detecting the operating system and CPU architecture
[INFO] ------------------------------------------------------------------------
[INFO] os.detected.name: linux
[INFO] os.detected.arch: x86_64
[INFO] os.detected.version: 4.15
[INFO] os.detected.version.major: 4
[INFO] os.detected.version.minor: 15
[INFO] os.detected.release: ubuntu
[INFO] os.detected.release.version: 18.04
[INFO] os.detected.release.like.ubuntu: true
[INFO] os.detected.release.like.debian: true
[INFO] os.detected.classifier: linux-x86_64
[INFO] 
[INFO] ---< com.amazon.opendistroforelasticsearch:opendistro_security_ssl >----
[INFO] Building Open Distro Security SSL 0.7.0.0
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- maven-clean-plugin:3.1.0:clean (default-clean) @ opendistro_security_ssl ---
[INFO] Deleting /opendistro/security-ssl/target
[INFO] 
[INFO] --- maven-enforcer-plugin:3.0.0-M2:enforce (enforce-maven) @ opendistro_security_ssl ---
[INFO] 
[INFO] --- maven-enforcer-plugin:3.0.0-M2:enforce (enforce-java) @ opendistro_security_ssl ---
[INFO] 
[INFO] --- git-commit-id-plugin:2.2.3:revision (get-the-git-infos) @ opendistro_security_ssl ---
[INFO] 
[INFO] --- maven-resources-plugin:3.1.0:resources (default-resources) @ opendistro_security_ssl ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 3 resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.8.0:compile (default-compile) @ opendistro_security_ssl ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 19 source files to /opendistro/security-ssl/target/classes
[INFO] -------------------------------------------------------------
[WARNING] COMPILATION WARNING : 
[INFO] -------------------------------------------------------------
[WARNING] bootstrap class path not set in conjunction with -source 8
[INFO] 1 warning
[INFO] -------------------------------------------------------------
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /opendistro/security-ssl/src/main/java/com/amazon/opendistroforelasticsearch/security/ssl/transport/OpenDistroSecuritySSLRequestHandler.java:[56,42] cannot find symbol
  symbol:   class NettyTcpChannel
  location: package org.elasticsearch.transport.netty4
[INFO] 1 error
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 2.883 s
[INFO] Finished at: 2019-03-18T16:27:44-03:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.0:compile (default-compile) on project opendistro_security_ssl: Compilation failure
[ERROR] /opendistro/security-ssl/src/main/java/com/amazon/opendistroforelasticsearch/security/ssl/transport/OpenDistroSecuritySSLRequestHandler.java:[56,42] cannot find symbol
[ERROR]   symbol:   class NettyTcpChannel
[ERROR]   location: package org.elasticsearch.transport.netty4
[ERROR] 
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
```
I just changed the value of elasticsearch.version to 6.6.1 in the pom.xml and plugin-descriptor.properties files.

Regards!

**Comment by HernanMora** [comment]
Hi,
Were you able to fix the error I sent you above? I've tried testing the plugin again and it's still not working for me.

Thanks

**Comment by hardik-k-shah** [comment]
HI @HernanMora ,

There is a tight coupling to Elasticsearch version with plugins. And we are planning to cut a new release for Open Distro with 6.6 support in a week or so.

Thanks!

**Comment by HernanMora** [comment]
Great, thanks for answering

**Comment by hardik-k-shah** [comment]
We have released V0.0.8.0 for ES 6.6.2 support https://github.com/opendistro-for-elasticsearch/security-ssl/releases/tag/v0.8.0.0
